test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js

```diff
@@ -125,20 +126,22 @@ test.requestHooks(mockWithoutIdentify)('should only render idp buttons with iden
     formName: 'redirect-idp',
   });
 
-  await t.expect(identityPage.identifierFieldExists('.o-form-input .input-fix input')).eql(false);
-  await t.expect(identityPage.getIdpButton('.social-auth-facebook-button').textContent).eql('Sign in with Facebook');
-  await t.expect(identityPage.getIdpButton('.social-auth-google-button').textContent).eql('Sign in with Google');
-  await t.expect(identityPage.getIdpButton('.social-auth-linkedin-button').textContent).eql('Sign in with LinkedIn');
-  await t.expect(identityPage.getIdpButton('.social-auth-microsoft-button').textContent).eql('Sign in with Microsoft');
+  await t.expect(identityPage.identifierFieldExistsForIdpView()).eql(false);
+  await t.expect(identityPage.getIdpButton('Sign in with Facebook').exists).eql(true);
+  await t.expect(identityPage.getIdpButton('Sign in with Google').exists).eql(true);
+  await t.expect(identityPage.getIdpButton('Sign in with LinkedIn').exists).eql(true);
+  await t.expect(identityPage.getIdpButton('Sign in with Microsoft').exists).eql(true);
 });
 
 test.requestHooks(logger, mockOnlyOneIdp)('should auto redirect to 3rd party IdP login page with basic Signing in message', async t => {
   await setup(t);
 
-  await checkConsoleMessages({
-    controller: null,
-    formName: 'success-redirect',
-  });
+  if(!userVariables.v3) {
+    await checkConsoleMessages({
+      controller: null,
+      formName: 'success-redirect',
```

Comment:
did not need to redirect to 'success-redirect' in V3

Reply:
I think we will need this in v3 (because customers hook on these form values). 

I think we may be missing some logic for this scenario, see: https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/responseTransformer.js#L146-L171

Reply:
ah okay yeah good point I will add this logic in